src/test/java/com/work/management/service/employee/impl/EmployeeServiceImplTest.java

```diff
@@ -60,5 +89,42 @@ private static Employee getEmployee() {
         .phoneNumber("9654092992").manager(3)
         .build();
   }
+
+  private static ImmutableList<Employee> getEmployeeList() {
+    return ImmutableList.of(
+        Employee.builder().firstName("Mudit").lastName("Tanwar").userName("Mudit.Tanwar")
+            .teamId("c7f6fbab-22fb-41bc-9300-0cc27c0de5c5").phoneNumber("9654092992").manager(3)
+            .build(),
+        Employee.builder().firstName("Ankit").lastName("Mathur").userName("Ankit.Mathur")
+            .teamId("c7f6fbab-22fb-41bc-9300-0cc27c0de5c6").phoneNumber("9654092990").manager(4)
+            .build(),
+        Employee.builder().firstName("Vishesh").lastName("Wason").userName("Vishesh.Wason")
+            .teamId("c7f6fbab-22fb-41bc-9300-0cc27c0de5c9").phoneNumber("9654092934").manager(5)
+            .build());
+  }
+
+  private static EmployeeDto getEmployeeDto(Employee employee) {
+    EmployeeDto employeeDto = new EmployeeDto();
+    BeanUtils.copyProperties(employee, employeeDto);
+    return employeeDto;
+  }
+
+  private static BulkEmployeeDto getBulkEmployeeDto() {
+    return BulkEmployeeDto.builder()
```

Comment:
This method could be inline, as it is only getting used once..... 
Do this everywhere, whichever method is getting used once and is also one liner, remove that method and inline the call.